posthog/tasks/org_usage_report.py

```diff
@@ -127,17 +127,19 @@ def send_all_reports(
             report: dict = {
                 **metadata,
                 **usage,
-                "organization_id": id,
+                "organization_id": organization_id,
                 "organization_name": org["name"],
                 "organization_created_at": org["created_at"],
                 "organization_user_count": org["user_count"],
                 "team_count": len(org["teams"]),
             }
             org_reports.append(report)  # type: ignore
         except Exception as err:
-            report_org_usage_failure(distinct_id, str(err))
-        if not (dry_run or settings.TEST or settings.DEBUG):
-            report_org_usage(distinct_id, report)
+            logger.warning("Organization usage report calculation failed", err)
+            if not dry_run:
+                report_org_usage_failure(organization_id, distinct_id, str(err))
+        if not dry_run:
```

Comment:
Curious why you no longer check `settings.TEST or settings.DEBUG`, seems like it still might be worth not to send the report if we're on debug/test mode.

Reply:
In tests, `dry_run` is also specified. I agree that keeping this behavior for `settings.DEBUG` would be helpful, though.

Reply:
In DEBUG we turn off tracking in posthog/apps.py